processing_scripts/Workflow7_ImageJ_Jython_Analyst.py

```diff
@@ -0,0 +1,236 @@
+#-----------------------------------------------------------------------------
+#  Copyright (C) 2017 University of Dundee. All rights reserved.
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+#------------------------------------------------------------------------------
+
+# This Jython script uses ImageJ to Subtract Background.
+# The purpose of the script is to be used in the Scripting Dialog
+# of Fiji.
+# Error handling is omitted to ease the reading of the script but this should be added
+# if used in production to make sure the services are closed
+# Information can be found at https://docs.openmicroscopy.org/omero/5.4.1/developers/Java.html
```

Comment:
I suppose I have to select ``java`` as language in Fiji script editor when running this script ? Please write this information here.

Reply:
No, wrong, I had to select ``Python``. It should have been written here I guess. There is no option ``Jython`` in the scripting dialog.